cengage/react-magma
# feat(treeview): add support for isDisabled

Issue: #1305

## What I did

New feature: add support `isDisabled` prop for TreeView to disable all items
New feature: add support `isDisabled` field for items of `preselectedItems` prop to manage disabled state for items

## Checklist 

## How to test
Storybook > TreeView > Complex:
We can see that item 'Section 5.1.2' is disabled by setting `isDisabled` to true via  preselectedItems
We can switch TreeView's `isDisabled` prop to true to disable all items
